dll/win32/winmm/mci.c

```diff
@@ -37,24 +37,32 @@
  * - command table handling isn't thread safe
  */
 
+#include <stdlib.h>
+#include <stdarg.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "windef.h"
+#include "winbase.h"
+#include "wingdi.h"
+#include "mmsystem.h"
+#include "winuser.h"
+#include "winnls.h"
+#include "winreg.h"
+#include "wownt32.h"
+
+#include "digitalv.h"
 #include "winemm.h"
 
-#include <mmsystem.h>
-#include <wownt32.h>
-#include <digitalv.h>
+#include "wine/debug.h"
```

Comment:
@TAN-Gaming Does this sync conflict with one of your WINMM PRs?

Reply:
No, I didn't make any change to this file so it's fine.